terraform/globals.tf

```diff
@@ -40,3 +40,12 @@ variable "office_cidrs" {
   default     = "80.194.77.90/32,80.194.77.100/32"
 }
 
+variable "dns_zone_id" {
+  description = "Amazon Route53 DNS zone identifier"
+  default = "Z3SI0PSH6KKVH4"
```

Comment:
a small nitpick, we align all these declarations so that `=` signs are exactly below each other